src/Tizen.AIAvatar/src/AIServices/Core/AIServiceData.cs

```diff
@@ -0,0 +1,185 @@
+﻿/*
+ * Copyright(c) 2024 Samsung Electronics Co., Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+
+
+namespace Tizen.AIAvatar
+{
+    /// <summary>
+    /// Provides data for LLM response events.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class llmResponseEventArgs : EventArgs
+    {
+        /// <summary>
+        /// Gets or sets the task ID associated with the LLM response.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public int TaskID { get; set; }
+
+        /// <summary>
+        /// Gets or sets the response text from the LLM.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public string Text { get; set; }
+
+        /// <summary>
+        /// Gets or sets the error message, if any, from the LLM response.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public string Error { get; set; }
+    }
+
+    /// <summary>
+    /// Provides data for TTS streaming events.
+    /// </summary>
+    [EditorBrowsable(EditorBrowsableState.Never)]
+    public class ttsStreamingEventArgs : EventArgs
+    {
+        /// <summary>
+        /// Gets or sets the audio data of the current chunk.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public byte[] AudioData { get; set; }
+
+        /// <summary>
+        /// Gets or sets the sample rate of the audio data.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public int SampleRate { get; set; }
+
+        /// <summary>
+        /// Gets or sets the text being converted to speech.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public string Text { get; set; }
+
+        /// <summary>
+        /// Gets or sets the voice used for the TTS conversion.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public string Voice { get; set; }
+
+        /// <summary>
+        /// Gets or sets the size of the current audio data chunk in bytes.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public int AudioBytes { get; set; }
```

Comment:
The variables AudioBytes, TotalBytes, and ProcessedBytes are all declared as 'int' because they represent the size of processed audio data. The actual audio data is stored in a variable named AudioData which is declared as a byte array.